packages/backend/src/open_payments/auth/middleware.test.ts

```diff
@@ -266,4 +316,145 @@ describe('Auth Middleware', (): void => {
     expect(ctx.grant).toEqual(grant)
     scope.isDone()
   })
+
+  test('returns 200 with valid http signature', async (): Promise<void> => {
+    Config.skipSignatureVerification = false
```

Comment:
I don't think this will suffice.
I think the auth middleware will use the value from when when we call:
```
    deps = await initIocContainer(Config)
    appContainer = await createTestApp(deps)
```
in the `beforeAll`

Reply:
I removed the `skipSignatureVerification` option entirely since existing tests should still pass